packages/algob/test/fixture-projects/default-config-project/assets/README.md

```diff
@@ -0,0 +1 @@
+This is a sample README file.
```

Comment:
Can we have info here?

Reply:
what kind of info? I can add whatever you want.

My intention was just to create some other file (which is not `.teal`) so I can test that only teal files are handled and compiled.

Reply:
Oh. Then this could be the contents of this file.